src/MoBi.Presentation/Presenter/EditSpatialStructurePresenter.cs

```diff
@@ -78,18 +78,21 @@ private void setInitialView()
 
       public override object Subject => _spatialStructure;
 
-      protected override (bool canHandle, IContainer containerObject) SpecificCanHandle(IObjectBase selectedObject)
+      protected override (bool canHandle, IContainer containerObject) SpecificCanHandle(EntitySelectedEvent entitySelectedEvent)
```

Comment:
This is the reason for all the various changes. The signature of this virtual method had to change so I could determine if the sender of the event was the subpresenter, and if so, handle the event.